client/src/main/scala/com/aliyun/emr/rss/client/write/LifecycleManager.scala

```diff
@@ -184,30 +187,30 @@ class LifecycleManager(appId: String, val conf: RssConf) extends RpcEndpoint wit
 
   override def receiveAndReply(context: RpcCallContext): PartialFunction[Any, Unit] = {
     case RegisterShuffle(applicationId, shuffleId, numMappers, numPartitions) =>
-      logDebug(s"Received RegisterShuffle request, " +
+      logInfo(s"Received RegisterShuffle request, " +
         s"$applicationId, $shuffleId, $numMappers, $numPartitions.")
       handleRegisterShuffle(context, applicationId, shuffleId, numMappers,
         numPartitions)
 
     case Revive(applicationId, shuffleId, mapId, attemptId, reduceId, epoch, oldPartition, cause) =>
-      logDebug(s"Received Revive request, " +
+      logInfo(s"Received Revive request, " +
```

Comment:
I'm afraid these logInfos will log too much. Maybe we should refactor the logs, distinguish Debug level from Trace level. These can be set to Debug level.

Reply:
> I'm afraid these logInfos will log too much. Maybe we should refactor the logs, distinguish Debug level from Trace level. These can be set to Debug level.

Change this to debug